cypress/features/regression/designSystem/toast.feature

```diff
@@ -5,22 +5,18 @@ Feature: Toast Design System component
     Given I open Design Systems default_story "Toast" component docs page
 
   @positive
-  Scenario: Verify the click action in Actions Tab
-    Given I open basic Test "Toast" component page
-    When clear all actions in Actions Tab
-      And I click closeIcon
-    Then click action was called in Actions Tab
+  Scenario: Verify a stacked Toast component
+    # commented because of BDD default scenario Given - When - Then
```

Comment:
commented lines are not necessary because you have Given and Then steps here. We need these comments when you have only Then step. Remove these comments, please